renderer/components/modal/modals/address.js

```diff
@@ -0,0 +1,48 @@
+import {useEffect, useState} from "react"
+import Modal from "../modal"
+import ImportKeys from "../../load/import_keys"
+import styles from "../../../styles/modal.module.css"
+import GetWallet from "../../util/wallet"
+import GetAddresses from "../../util/addresses"
+
+const AddressModal = ({onClose, setLastUpdate}) => {
+    const [error, setError] = useState("")
+    const onSetKeysAndAddresses = async (keys, addresses) => {
+        const wallet = await GetWallet()
+        if((keys.length > 0) && wallet.keys.length == 0){
+            console.log("Error, cannot add keys to a keyless wallet")
+            setError("Error, cannot add keys to a keyless wallet")
+            return
+        }
+        else if((addresses.length > 0) && wallet.keys.length > 0){
```

Comment:
Styling I've been using for this repo would be same line with spacing:
```
} else if ((addresses.length > 0) && wallet.keys.length > 0) {
```
e.g. https://github.com/memocash/desktop/blob/master/renderer/components/modal/modals/profile/post_like.js#L34

I try not to be a stickler on styling, but will mention it from time to time.